Phonebook/phonebook/requirements.yaml

```diff
@@ -1,6 +1,6 @@
 dependencies:
   - name: traefik
     condition: traefik.enabled
-    version: 1.86.2
-    appVersion: 1.7.14
-    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 3.4.0
```

Comment:
does the yml support wildcards for version numbers? so we can do something like `3.4.x` or `3.x.x`. So we get the security fixes faster?

Reply:
nope